config/initializers/courses.rb

```diff
@@ -0,0 +1,76 @@
+module Courses
+  DEFINITIONS = [
+    {
+      position: 0,
+      name: "Additional Support Offer for new headteachers",
+      ecf_id: "7fbefdd4-dd2d-4a4f-8995-d59e525124b7",
+      identifier: "npq-additional-support-offer",
+      description: "The Additional Support Offer is a targeted support package for new headteachers.",
+      display: false,
+    },
+    {
+      position: 1,
+      name: "NPQ for Leading Behaviour and Culture (NPQLBC)",
```

Comment:
These names are all still the old names. They aren't used anywhere in the interface however they are used in emails and the admin interface. To avoid making unexpected changes I've left the names as is, then we can return after and replace those calls to `.name` with separate localisations and remove the name/description columns entirely.